examples/files.json

```diff
@@ -120,7 +120,6 @@
 		"webgl_loader_texture_tga",
 		"webgl_loader_ttf",
 		"webgl_loader_vox",
-		"webgl_loader_vrm",
 		"webgl_loader_vrml",
```

Comment:
@mrdoob you missed the spot

Reply:
Hmm?

Reply:
you removed webgl_loader_vrm with the anime girl but not webgl_loader_vrml with the building

Reply:
wait, are these actually two different loaders 🤦

Reply:
```js
VRMLoader
VRMLLoader
```

Spot the difference 😁